bin/sync_django_ldap.py

```diff
@@ -99,54 +92,50 @@ def main():
                 os.setgid(apache_gid)
                 os.setuid(apache_uid)
 
-                _log.info("Now running as %s" % (os.geteuid(),))
+                logging.info("Now running as %s" % (os.geteuid(),))
             except OSError:
-                _log.raiseException("Could not drop privileges")
+                logger.raiseException("Could not drop privileges")
 
             client = AccountpageClient(token=opts.options.access_token, url=opts.options.account_page_url + '/api/')
             syncer = LdapSyncer(client)
-            last = int((datetime.datetime.strptime(last_timestamp, "%Y%m%d%H%M%SZ") -
-                       datetime.datetime(1970, 1, 1)).total_seconds())
+            last = int(
+                (datetime.datetime.strptime(last_timestamp, "%Y%m%d%H%M%SZ") -
+                datetime.datetime(1970, 1, 1)).total_seconds())
```

Comment:
add `tzinfo=utc` (see https://github.com/hpcugent/vsc-utils/blob/master/lib/vsc/utils/timestamp.py#L82)

Reply:
hmmm, or not. i guess the first datetime from strptime also has no timezoneinfo, better not to specify it at all